packages/airnode-adapter/src/types.ts

```diff
@@ -62,7 +62,8 @@ export type BaseResponseType = typeof baseResponseTypes[number];
 // Use might pass a complex type (e.g. int256[3][]) which we cannot type
 export type ResponseType = string;
 
-export interface ReservedParameters {
+// Reserved parameters specific for response processing i.e. excludes _gasPrice
+export interface ResponseReservedParameters {
```

Comment:
I renamed `ReservedParameters` to `ResponseReservedParameters` to distinguish those that are relevant to response processing (`_type`, `_path`, and `_times`) from those that are not i.e. `_gasPrice` and (coming soon) `_minConfirmations`